db/migrations/20201109145014_add_feedbacks_table.js

```diff
@@ -0,0 +1,22 @@
+const { addUpdateTsTrigger } = require("../util")
+
+exports.up = async (knex) => {
+  await knex.schema.createTable("feedbacks", (table) => {
```

Comment:
Let's just name this `feedback`, which is both the singular and plural

Reply:
"feedback" is indeed plural, but we're talking here about rows/objects. I think that using "feedback" might confuse future maintainers.
(I also think that all the table names should be singular, but that's a different discussion)